dev/MRTCore/azure-pipelines.yml

```diff
@@ -215,10 +174,10 @@ jobs:
   variables:
     solution: '**/*.sln'
     appxPackageDir: '$(build.artifactStagingDirectory)\AppxPackages\\'
-    codeSign: false
+    codeSign: ${{ eq(variables['System.definitionId'], '57248') }}
```

Comment:
pretty sure we are going to sign everything at the Project Reunion (top) level en masse ...  your script should probably (eventually) just build and copy binaries to an agreed upon aggregation point.  I think Geoffrey is working on this right now.

Reply:
sure. until then I'd still maintain this one.

Reply:
do we know where is the aggregation point?

Reply:
I think the aggregation point will just be some place in the ArtifactStagingDirectory, almost exactly what you're doing now.  I think all we will need to do is change the artifact name from "mrt" to something common that all components can use.